pages/phases/job-analysis/defining-proficiency-levels.md

```diff
@@ -6,7 +6,7 @@ title: Defining Proficiency Levels and Required Experience
 sidenav: hiring-phases
 phase: job-analysis
 sub-phase-order: 050
-intro: Once you've defined your competencies, define the proficiency levels you'll use to compare an applicant's experience against the competencies. This establishes the required experience for the position.
+intro: Define the proficiency levels SMEs will use to compare an applicant's experience against your competencies. This establishes the specialized experience for the position.
```

Comment:
the competnices and proficiencies are specialized experience; not just the proficiencies.